### PR TITLE
[openexr] Initial integration of OpenEXR fuzzers.

### PR DESCRIPTION
**Pull request by alsophian**
This commit adds a working `build.sh` to build a basic OpenEXR fuzzer from https://github.com/google/security-research-pocs.  I lazily just used an `ADD` command in the Dockerfile to get one fuzzer from there; you'll probably want to import that fuzzer directly, and perhaps add a few more.

### CODE REVIEW

**Comment by alsophian** [comment]
Wait, this PR is open against google/oss-fuzz instead of rjotwani/oss-fuzz?  Man, I am not very good at GitHub, sorry.

File: projects/openexr/project.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+homepage: "https://openexr.com"
+language: c++
+primary_contact: "twodeecoda@gmail.com"
```
Comment (Dor1s) [inline projects/openexr/project.yaml:3]: do we have any confirmation / approval from the maintainers?

File: projects/openexr/openexr_deeptiles_fuzzer.cc [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  delete file;
+}
+
+}  // namespace
+
+// from cl/164883104
+static char *buf_to_file(const char *buf, size_t size) {
+  char *name = strdup("/dev/shm/fuzz-XXXXXX");
```
Comment (Dor1s) [inline projects/openexr/openexr_deeptiles_fuzzer.cc:166]: this isn't recommended. Instead we suggest to use an in memory file if possible

File: projects/openexr/project.yaml (hunk 1 above)
Comment (alsophian) [reply projects/openexr/project.yaml:3]: Yes, I can forward you the email thread if you like.

File: projects/openexr/openexr_deeptiles_fuzzer.cc (hunk 2 above)
Comment (alsophian) [reply projects/openexr/openexr_deeptiles_fuzzer.cc:166]: @rjotwani and I are working on a second PR that modifies these fuzzers to use OpenEXR's `Imf::IStream` class for in-memory input.  We can merge that PR with this one if you'd prefer that over reviewing it separately.

**Pull request merged**